src/test/java/uk/gov/pay/api/utils/mocks/ConnectorMockClient.java

```diff
@@ -77,6 +77,7 @@ private String buildChargeResponse(long amount, String chargeId, PaymentState st
                 .add("card_brand", CARD_BRAND_LABEL)
                 .add("created_date", createdDate)
                 .add("language", language.toString())
+                .add("delayed_capture", Boolean.toString(delayedCapture))
```

Comment:
If it’s a `Boolean` object, you can just `toString()` the object directly: `delayedCapture.toString()`.

This works if it’s a primitive though.

Reply:
But, of course, as demonstrated in your latest code, we actually want the value itself, not its stringification!